### PR TITLE
finished challenge

### PR DESCRIPTION
**Pull request by Nhuuu**

### CODE REVIEW

**Comment by Nhuuu** [comment]
- [ x] Top-level README “Table of Contents” is updated
 - [x ] Feature tasks for this challenge are completed
 - [ x] Unit tests written and passing
     - [x ] “Happy Path” - Expected outcome
     - [ x] Expected failure
     - [ x] Edge Case (if applicable/obvious)
 - [x ] README for this challenge is complete
     - [ x] Summary, Description, Approach & Efficiency, Solution
     - [x ] Link to code
     - [x ] Picture of whiteboard

File: Data-Structures/src/main/java/tree/BinaryTree.java [hunk 1]
```diff
@@ -62,22 +62,31 @@ public BinaryTree(Node<T> root){
...
-        q.enqueue(current.right.value);
+        q.enqueue(current.right);
       }
-      ans.add(q.dequeue());
     }
     return ans;
   }
 
+  public int findMaxValue(Node root){
+    int max = -1;
```
Comment (evanslaton) [inline Data-Structures/src/main/java/tree/BinaryTree.java:84]: Food for thought: will the user know `-1` means the tree is empty or think that that is the max value in the tree?
